source/core/src/main/com/csse3200/game/components/tractor/TractorActions.java

```diff
@@ -282,9 +282,9 @@ private Array<Object> getTilesHarvest(String dir) {
    */
   private void harvest(TerrainTile tile) {
     if (tile == null) {
-      return;
```

Comment:
what is sonarcloud smoking

Reply:
There is unnecessary jumps, wasting those 2 clock cycles from the rjmp's, it is a if-else cascade for all so won't fall into others anyway